doc/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "vuepress build",
-    "dev": "NODE_ENV=development vuepress dev"
+    "dev": "vuepress dev"
```

Comment:
Do we not need this anymore? If not, let's remove the use in the `.vuepress/config.js`

Reply:
It doesn't work on Windows, and I got tired of removing it

Reply:
It only affects the external link to the API reference. I will remove it from the config